components/MeetingDetail/MeetingDetail.test.js

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from "@testing-library/react";
+import MeetingDetail from "./MeetingDetail";
+import { data } from "../../data";
```

Comment:
I recommend declaring a test variable for data in this very file; by doing so, every developer (including future-Alex) will now what kind of data is passed to the tested component.